src/main/java/baritone/process/MineProcess.java

```diff
@@ -467,8 +469,9 @@ public void mineByName(int quantity, String... blocks) {
     @Override
     public void mine(int quantity, BlockOptionalMetaLookup filter) {
         this.filter = filter;
-        if (filter != null && !Baritone.settings().allowBreak.value) {
-            logDirect("Unable to mine when allowBreak is false!");
+//        baritone.getPathingBehavior().updateIsCloseToHome();
+        if (filter != null && (!Baritone.settings().allowBreak.value || baritone.getPathingBehavior().isCloseToHome())) {
+            logDirect("Unable to mine when allowBreak is false or when you\'re too close to home!");
```

Comment:
This should as well be removed depending on when home protection is checked for pathing.